charts/spire/README.md

```diff
@@ -191,6 +191,8 @@ Now you can interact with the Spire agent socket from your own application. The
 | spiffe-oidc-discovery-provider.insecureScheme.nginx.image.tag | string | `"1.24.0-alpine"` | Overrides the image tag |
 | spiffe-oidc-discovery-provider.insecureScheme.nginx.image.version | string | `""` | This value is deprecated in favor of tag. (Will be removed in a future release) |
 | spiffe-oidc-discovery-provider.insecureScheme.nginx.resources | object | `{}` |  |
+| spiffe-oidc-discovery-provider.jobAnnotations."helm.sh/hook" | string | `"pre-delete"` |  |
+| spiffe-oidc-discovery-provider.jobAnnotations."helm.sh/hook-delete-policy" | string | `"before-hook-creation, hook-succeeded, hook-failed"` |  |
```

Comment:
can these be updated to be jobPreDeleteAnnotations in case we have new types of jobs in the future?

Reply:
What do you think of putting under the existing `deleteHook` and `installAndUpgradeHook` configurables?